blessclient/client.py

```diff
@@ -20,7 +20,13 @@
 from random import randint
 
 import awsmfautils
-import tokengui
+
+ENABLE_GUI=True
+try:
+    import tokengui
+except ImportError:
+    ENABLE_GUI=False
```

Comment:
I took roughly the same approach in my branch, a (slight) improvement would be:

https://github.com/lyft/python-blessclient/pull/25

```python
try:
    from . import tokengui
except ImportError:
    tokengui = None
```

and

```python
if showgui and tokengui:
    ...
```